container/docker/factory.go

```diff
@@ -46,6 +46,8 @@ var dockerRunDir = flag.String("docker_run", "/var/run/docker", "Absolute path t
 // --cgroup-parent have another prefix than 'docker'
 var dockerCgroupRegexp = regexp.MustCompile(`.+-([a-z0-9]{64})\.scope$`)
 
+var noSystemd = flag.Bool("nosystemd", false, "Don't assume systemd is being used by docker.")
```

Comment:
How about something like `Explicitly disable  systemd support for Docker cgroups`?